composer.json

```diff
@@ -26,7 +26,8 @@
         "symfony/monolog-bundle"               : "~2.7",
         "symfony/swiftmailer-bundle"           : "~2.3",
         "symfony/symfony"                      : "~2.8",
-        "twig/extensions"                      : "~1.2"
+        "twig/extensions"                      : "~1.2",
+        "lexik/form-filter-bundle": "dev-master"
```

Comment:
depending on dev-master is a no-go. If the bundle does not have stable releases (which should be considered though), a branch alias should at least be used to avoid getting bumped to a new major version if they start developing it. If they don't support installing it the bundle other than with a ``dev-master`` requirement, I vote against including it in the demo.
And if they don't have a branch alias, you should open a PR adding it.